app/javascript/app/components/disclaimer/disclaimer.js

```diff
@@ -0,0 +1,26 @@
+import { createElement, PureComponent } from 'react';
+import Component from './disclaimer-component';
+
+class DisclaimerContainer extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      hasBeenShown: JSON.parse(localStorage.getItem('disclaimerShown'))
```

Comment:
Could we move this to a constant?

Reply:
A constant won't update the component with the createElement like the state does. If you think that it could let me know and we can fix it tomorrow